src/site/content/en/blog/wix/index.md

```diff
@@ -0,0 +1,393 @@
+is sent by our servers to the client, without a body.
+
+<figure class="w-figure">
+  {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/hr1xnQccJEkHTBGxS5wF.jpg", alt="ALT_TEXT_HERE", width="800", height="410" %}
+  <figcaption class="w-figcaption">WebPageTest Repeat View</figcaption>
+</figure>
+
+In addition, this change means that our HTML is no longer visitor-specific and
+contains no cookies. In other words it can basically be cached anywhere, opening
+the door to using CDN providers that have much better geo presence in hundreds
+of locations around the world.
+
+
+### DNS, SSL and HTTP/2
+
+With caching enabled, wait times were reduced and other important parts of the
+initial connection became more substantial. Enhancing our networking
+infrastructure and monitoring enabled us to improve our DNS, connection, and SSL
+times.
+
+<figure class="w-figure w-figure--inline-right">
+  {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/Uuvh9xTItQ8wMA9s13RP.jpg", alt="A response time graph.", width="800", height="441" %}
+</figure>
+
+[HTTP/2](https://en.wikipedia.org/wiki/HTTP/2) was enabled for all user domains,
+reducing both the amount of connections required and the overhead that comes
+with each new connection. This was a relatively easy change to deploy, while
+taking advantage of the [performance and resilience
+benefits](https://developers.google.com/web/fundamentals/performance/http2) that
+come with HTTP/2.
+
+
+### Brotli compression (vs. gzip)
+
+<div class="w-figure">
+  <div class="w-stats">
+    <div class="w-stat">
+     <p class="w-stat__figure">21 - 25<sub class="w-stat__sub">%</sub></p>
+     <p class="w-stat__desc">Reduction of median file transfer size</p>
+    </div>
+  </div>
+</div>
+
+Traditionally, all our files were compressed using [gzip
+compression](https://en.wikipedia.org/wiki/Gzip), which is the most prevalent
+HTML compression option on the web. This compression protocol was initially
+implemented almost 30 years ago!
+
+<figure class="w-figure w-figure--inline-right">
+  {% Img src="image/BrQidfK9jaQyIHwdw91aVpkPiib2/h7KzeAfg2THMdDGMYvlR.jpg", alt="Brotli compression", width="800", height="805" %}
+  <figcaption class="w-figcaption">
+    <a href="https://tools.paulcalvano.com/compression.php">Brotli Compression Level Estimator</a>
+  </figcaption>
+</figure>
+
+
+The newer [Brotli compression](https://en.wikipedia.org/wiki/Brotli) introduces
+compression improvements with almost no trade-offs, and is slowly becoming more
+popular, as described in the yearly Web Almanac [Compression
+chapter](https://almanac.httparchive.org/en/2020/compression#what-type-of-content-should-we-compress).
+It has been supported by [all the major browsers](https://caniuse.com/brotli)
+for a while.
+
+We enabled Brotli support on our nginx proxies in the edges, for all clients
+that support it.
+
+Moving to use Brotli compression reduced our median file transfer sizes by **21%
```

Comment:
Can't break line here - it breaks the bold of 21%-25% (see in deploy preview)

Reply:
Sharp eye, thanks for catching